src/main/java/pixy/tasks/Deadlines.java

```diff
@@ -0,0 +1,55 @@
+package pixy.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a Task object with a specified due date and time.
+ */
+public class Deadlines extends Task {
+
+    /** Due date of the given Task*/
+    private LocalDateTime dueDateTime;
+
+    /**
+     * Creates a Deadline object with specified due date.
+     *
+     * @param description The description of the task.
+     * @param by The due date and time of the task.
+     */
+    public Deadlines(String description, String by) {
+        super(description);
+        DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+        DateTimeFormatter displayFormatter = DateTimeFormatter.ofPattern("MMM d yyyy, h:mm a");
```

Comment:
Perhaps you should make these two variables constants. 😄

Reply:
i agree, perhaps that will make the code look cleaner